bundles/ch.elexis.core.l10n/src/ch/elexis/core/l10n/messages_de.properties

```diff
@@ -6831,6 +6831,10 @@ Patientenblatt2_showAddress = Adresse zeigen...
 
 Patientenblatt2_showBezugKontaktRelation = Kontaktbezug zeigen...
 
+PatientenStamm=Patienten Stamm Blueconnect
```

Comment:
Wollen wir diese Übersetzungen wirklich im core haben? Das sieht für mich sehr spezifisch für diese Auswertung aus, die sich im medelexis repository befindet.

Reply:
Ich bin hier auch unsicher. Ich schlage vor, dass wir Strings, welche wir NUR im medelexis-3 benutzen, dort lassen und nicht in den elexis-3-core bringen.

Reply:
@jcolameo kannst du bitte die Übersetzungen wieder in das archie bundle verschieben, und diesen pull request schliessen?